webpack.config.devtools.js

```diff
@@ -40,7 +41,13 @@ module.exports = webpackConfig => {
         callback(null, "var {}");
         return;
       } else if(nativeMapping[mod]) {
-        callback(null, "var devtoolsRequire('" + nativeMapping[mod] + "')");
+        const mapping = nativeMapping[mod];
+        if(Array.isArray(mapping)) {
+          callback(null, `var devtoolsRequire("${mapping[0]}")["${mapping[1]}"]`);
+        }
+        else {
+          callback(null, `var devtoolsRequire("${mapping}")`);
+        }
```

Comment:
Magic everywhere! 😈 I don't love it; hoping we can figure out a better way to share modules.